lib/src/posthog_config.dart

```diff
@@ -143,11 +143,58 @@ class PostHogErrorTrackingConfig {
   /// - All other packages are inApp unless in inAppExcludes
   var inAppByDefault = true;
 
+  /// Enable automatic capture of unhandled exceptions
```

Comment:
we could add one more config here for android autocapture https://github.com/PostHog/posthog-android/blob/3b565baef8b46510ca515a13f211e245edf1b4c6/posthog/src/main/java/com/posthog/errortracking/PostHogErrorTrackingConfig.kt#L16